### DISCUSSION TITLE
how do u get extension ids?

### DISCUSSION THREAD

**Discussion by SevBlzy** [3kh0/ext-remover#370] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
how do u get cisco umbrella and ghoper buddys  extenssion ids???

**Comment by ghost** [comment, marked as answer]
[Screen recording 2023-01-04 12.47.51 PM.webm](https://user-images.githubusercontent.com/77454237/210617840-eea13178-1d96-4b4a-a5d9-7e769757f351.webm)

**Comment by Aka-but-nice** [comment]
But explain why you need them, are you trying to do the exploit

**Reply by SevBlzy** [reply]
yeahi am doing the one where u type it into the box

**Comment by Alex2Mayhem** [comment]
go to chrome://extensions
Click on details on the extensions you want. 
At the URL Bar it will say. chrome://extensions/?id={bunch of randomlettersisyourid}
Copy whats after ?id=
